src/humanPose/rebaScore.js

```diff
@@ -434,13 +444,139 @@ function wristReba(rebaData) {
+    const leftUpperarmDirection = rebaData.joints[JOINTS.LEFT_SHOULDER].clone().sub(rebaData.joints[JOINTS.LEFT_ELBOW]).normalize();
+
+    // check if wrist position is outside +-15 deg, then +1 
+    let wristPositionAngle = angleBetween(leftHandDirection, leftForearmDirection);
+    if (wristPositionAngle > 15) {
+        leftWristScore += 1;
+    }
+
+    // check if the hand is bent away from midline, then +1
+    // the angle limit from midline is not specified in REBA definition (chosen by us)
+    let wristBendAngle = 90 - angleBetween(leftHandPinky2Index, leftForearmDirection);
+    if (Math.abs(wristBendAngle) > 30) {
+        leftWristScore += 1;
+    }
+
+    // check if the hand is twisted (palm up), then +1
+    const leftElbowAxis = new THREE.Vector3(); 
+    leftElbowAxis.crossVectors(leftUpperarmDirection, leftForearmDirection).normalize();
+    let wristTwistAngle = angleBetween(leftElbowAxis, leftHandPinky2Index);
+    /* disable for now because pose tracking does not capture hand twists
+    if (wristTwistAngle > 90) {
+        leftWristScore += 1;
+    }
+    */
+
+    leftWristScore = clamp(leftWristScore, 1, 3);
+
+    //console.log(`Left wrist: wristPositionAngle=${wristPositionAngle.toFixed(0)};  wristBendAngle=${wristBendAngle.toFixed(0)}; wristTwistAngle=${wristTwistAngle.toFixed(0)} deg`);
+
+    /* right wrist */
+    // compute main direction vectors
+    const rightPinky = rebaData.joints[JOINTS.RIGHT_PINKY].clone().sub(rebaData.joints[JOINTS.RIGHT_WRIST]).normalize();
+    const rightIndex = rebaData.joints[JOINTS.RIGHT_INDEX].clone().sub(rebaData.joints[JOINTS.RIGHT_WRIST]).normalize();
+    const rightHandDirection = rightPinky.clone().add(rightIndex).normalize();
+    const rightHandPinky2Index = rebaData.joints[JOINTS.RIGHT_INDEX].clone().sub(rebaData.joints[JOINTS.RIGHT_PINKY]).normalize();
+    const rightForearmDirection = rebaData.joints[JOINTS.RIGHT_WRIST].clone().sub(rebaData.joints[JOINTS.RIGHT_ELBOW]).normalize();
+    const rightUpperarmDirection = rebaData.joints[JOINTS.RIGHT_SHOULDER].clone().sub(rebaData.joints[JOINTS.RIGHT_ELBOW]).normalize();
+
+    // check if wrist position is outside +-15 deg, then +1 
+    wristPositionAngle = angleBetween(rightHandDirection, rightForearmDirection);
+    if (wristPositionAngle > 15) {
+        rightWristScore += 1;
+    }
+
+    // check if the hand is bent away from midline, then +1
+    // the angle limit from midline is not specified in REBA definition (chosen by us)
+    wristBendAngle = 90 - angleBetween(rightHandPinky2Index, rightForearmDirection);
+    if (Math.abs(wristBendAngle) > 40) {
+        rightWristScore += 1;
+    }
+
+    // check if the hand is twisted (palm up), then +1
+    const rightElbowAxis = new THREE.Vector3(); 
+    rightElbowAxis.crossVectors(rightUpperarmDirection, rightForearmDirection).normalize();
+    wristTwistAngle = angleBetween(rightElbowAxis, rightHandPinky2Index);
```

Comment:
see github actions comment, could put this line in the commented out block below